requirements-dev.txt

```diff
@@ -30,7 +30,9 @@ pre-commit==1.17.0
 psycopg2-binary==2.8.5
 pycodestyle==2.5.0
 pydruid==0.6.1
-pyhive==0.6.2
+# Enable in-place multirow inserts
+# TODO(bkyryliuk): release new version of pyhive
+git+https://github.com/dropbox/PyHive@master
```

Comment:
I think this will be a blocker - the community has not been accepting of installing dependencies from Github.